project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/components/_group_dashboard.scss

```diff
@@ -51,3 +51,7 @@
     width: 2em;
   }
 }
+
+.highlight-link {
```

Comment:
This affects the group dashboard and the project dashboard too. I thought it was better to leave it alone than duplicating it on both files. What change should I do?